samples/chat/README.md

```diff
@@ -0,0 +1,295 @@
+---
+name: ChatGPT + Enterprise data
+description: Chat with your data using OpenAI and AI Search.
+languages:
+- python
+- typescript
+- bicep
+- azdeveloper
+products:
+- azure-openai
+- azure-cognitive-search
+- azure-app-service
+- azure
+page_type: sample
+urlFragment: azure-search-openai-demo
+---
+<!-- Original document: /readme.md -->
+
+# ChatGPT + Enterprise data with Azure OpenAI and AI Search
+
+The [azure-search-openai-demo](/) project can set up a full RAG chat app on Azure AI Search and OpenAI so that you can chat on custom data, like internal enterprise data or domain-specific knowledge sets. For full instructions on setting up the project, consult the [main README](/README.md), and then return here for detailed instructions on the data ingestion component.
```

Comment:
Since this is the fully replicated README, I guess this doesn't need a prelim paragraph? Assuming all links works.